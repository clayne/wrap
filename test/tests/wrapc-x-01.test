wrapc | /dev/null | -x | wrapc-x-01.txt | 0
