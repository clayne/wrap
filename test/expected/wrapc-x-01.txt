/**
 * @brief Brief description.
 *
 * Detailed description.
 * @code
 * size_t example = foo( s, strlen( s ) );
 * if ( example > 0 ) {
 *     // ...
 * }
 * @endcode
 * Some other line 1.  Some other line 2.
 * @copydoc bar()
 *
 * @param s The null-terminated string to do something with.
 * @param s_len The length of \a s.
 * @return Returns the new length of \a s.
 */
size_t foo( char *s, size_t s_len );
